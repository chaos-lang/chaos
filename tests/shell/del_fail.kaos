num b = 5
del b[1]
b = 4
print b
num a = 3
print a
del b
exit
