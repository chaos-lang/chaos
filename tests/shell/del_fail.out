Chaos Error:
Module: ~/chaos/__interactive__.kaos
Line: 2
Unrecognized complex data type: Number for variable: b
Absorbed by Interactive Shell
4
3
[start] =>
	{id: 2, name: a, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 1, role: 0, param_of: } =>
[end]
You have chosen the order!
